### ISSUE TITLE
Unzalgo to a point?

### ISSUE THREAD

**Issue by Qix-** [kdex/unzalgo#35]
Hi there, is there a way to un-Zalgo up to a specific point?

For example, this text is really zalgo'd:<br /><br /><br /><br /><br />

S̷̡̡̡̱̦̣̹̭̻͔̣͖̤̜̮̓̒̋̋͌̄̊̄̎̓o̷̡̢̨̗͔̤̫͙̖̙͉̲̘͙͔͖̤͎̙͓̳̣̳̣͋̀̇́̈̏̾̇̇̀̎̔̓̇̆͝ḿ̸̨̮̟̣̙̮̲̭͎͓͖̘̒̀͌͆͊̿̾̄̽̀̔̈́̍̒͒̔̕͝e̷̛͖̤͍̬͖̔̈̆̐͌̃̓͌̽͑̾̐̇̑̇̈̂̋̂͠ ̵̢̨̞͕̥̯̼͈̺̖̞̥̳̤̓̇̓̓̈͆z̷̡̬̱̺̘̹̭̙̭͚̼̝̤̳̦̲̬̜͌͌̊̆ͅͅa̷̢̡̺͕͈̰̮̲͔̙̱̼͉̲̼̝̝̻̹̱̹̝̗̿̋͐̊͑͑͐̽̆̉̓͋̽̅̈́̚͜͝l̶͇͍͈̞̠̜͕͒̑͆̇̊̚͝g̸̨̛͎͚͚̗̘͙͔͓̠̝͔̬̳̗̯̮͍̻̥̃͊̏̐̌͒̀̓͛͠ȭ̷̢̡̧̤͇̮͕̘̱̬̖̪͈̘̟̉͑͌̑̿̇̊̿͛͊̎͌̀̽͛͋̃̑́̈́̈́̅͠ ̷̛͙͙̜̫̼̙̯́̉̊̿̈́́̽͛̓̓̊̓̋̏̀͌͠ͅt̷̠̞̯̤̃̇̒̾͒̑͋̒̈́͋͗̉̉͐̍̾͑̈́̈́͌͆̀̂̋͌͜ȩ̶̧̢̡̛̛̣͕̥͕͇̖͈̗͍̖̠͚̮͙̅̂̌́̐͛͗̽͋́̿͂̅̒͌̐͆̏̕͜͝͝͝͠x̶̡̧̛͚̗̖̙͚͍̻̙̥͓͖͕͍̮͖̙̙̜͓͈̩̯̐͛̏̍́͌̏̂̀̐͛͂̈́̆́̀̒̉̾̈́͌͘͜͠t̸̨̨̲̟͎̩̱̹̬͙̩̠͇̪͒̃̒͛̍̎̂͒̀́́̍.̵̮͐̋̐͐̅̿̿́̊͑́̂͗͂̊̽̚̕̕͘̕͘͠

<br /><br /><br /><br />
Could it be possible to instead trim the characters down to a certain 'height', instead of _entirely_ removing all of the combining characters?

S̶̫͛o̷̩̍m̸̰̐e̴̲͑ ̷̺̃z̴͕̾â̶̜l̶̗͝g̷̹̓ö̸̠́ ̶̨̈́t̶̠̑e̵̜̅x̷̨̃t̷̮̐.̵̰̊

Thanks for any input :)

**Comment by Qix-** [comment]
You're a beast, thank you so much.

**Comment by kdex** [comment]
Thanks for the interesting idea! I've just released this as `unzalgo@^3.0.0`. Note that this was a breaking change, so I've incremented the major version.

You can find the documentation [here](https://github.com/kdex/unzalgo/tree/b64881f86d824fe049d30db9b14d7bfd44899430#cleanstring-options-string-default-export).

Example:

```js
import { clean } from "unzalgo";
const cleaned = clean(someString, {
	targetDensity: 0.5
});
